otx/algorithms/segmentation/adapters/mmseg/models/losses/cross_entropy_loss_with_ignore.py

```diff
@@ -3,61 +3,65 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+from typing import Optional
+
 import torch
 import torch.nn.functional as F
 from mmseg.models.builder import LOSSES
-from mmseg.models.losses.utils import get_class_weight
-
-from .otx_pixel_base import OTXBasePixelLoss
+from mmseg.models.losses import CrossEntropyLoss
+from mmseg.models.losses.utils import weight_reduce_loss
 
 
 @LOSSES.register_module()
-class CrossEntropyLossWithIgnore(OTXBasePixelLoss):
-    """CrossEntropyLossWithIgnore with Ignore Mode Support for Class Incremental Learning.
+class CrossEntropyLossWithIgnore(CrossEntropyLoss):
+    """CrossEntropyLossWithIgnore with Ignore Mode Support for Class Incremental Learning."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._loss_name = "loss_ce_ignore"
+
+    def forward(
+        self,
+        cls_score: Optional[torch.Tensor],
+        label: Optional[torch.Tensor],
+        weight: Optional[torch.Tensor] = None,
+        avg_factor: Optional[int] = None,
+        reduction_override: Optional[str] = "mean",
+        ignore_index: int = 255,
+        valid_label_mask: Optional[torch.Tensor] = None,
+        **kwargs
+    ):
+        """Forward."""
```

Comment:
Same with above, to understand this function, more detailed docstring is needed.
I couldn't totally understand avg_factor, valid_label_mask.

Reply:
I updated, and avg_factor used in default args in cr loss.